test/wsfed.tests.js

```diff
@@ -94,6 +99,34 @@ describe('wsfed', function () {
     });
   });
 
+  describe('when using a different name identifier format', function (){
```

Comment:
Maybe we can rename this test suite to something like `when a name identifier format is passed as an auth option` (or similar) to be more explicit of the source of the name identifier format?